pages/linux/iotop.md

```diff
@@ -0,0 +1,32 @@
+# iotop
+
+> Display a table of current I/O usage by processes or threads.
+> More information: <https://linux.die.net/man/1/iotop>.
+
+- Start top-like I/O monitor:
+
+`iotop`
+
+- Only show processes or threads actually doing I/O:
+
+`iotop -o`
+
+- Show I/O usage in non-interactive mode:
+
+`iotop -b`
+
+- Only I/O usage of processes(Normally iotop shows all threads):
```

Comment:
Please rephrase this to be a verb. I'm unsure of what this actually does so I'm unable to give you a suggestion on how to do it haha.